L1Trigger/DTTriggerPhase2/plugins/DTTrigPhase2Prod.cc

```diff
@@ -866,4 +878,89 @@ void DTTrigPhase2Prod::processDigi(std::queue<std::pair<DTLayerId, DTDigi>>& inQ
   vec.push_back(&newQueue);
 }
 
+void DTTrigPhase2Prod::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  // dtTriggerPhase2PrimitiveDigis
+  edm::ParameterSetDescription desc;
+  desc.add<edm::InputTag>("digiTag", edm::InputTag("CalibratedDigis"));
+  desc.add<int>("timeTolerance", 999999);
+  desc.addUntracked<std::string>("geometry_tag", "");
+  desc.addUntracked<int>("trigger_with_sl", 4);
+  desc.addUntracked<double>("tanPhiTh", 1.0);
```

Comment:
Moreover: it is rather strange how do you define the parameters as "tracked" or "untracked": according to the names (even if not used, see previous comment...) most of those parameters are relevent for the algorithm, and they should be rather made "tracked"